cmd/metrics.go

```diff
@@ -72,6 +72,27 @@ func (c *minioCollector) Collect(ch chan<- prometheus.Metric) {
 	}
 	s := objLayer.StorageInfo(context.Background())
 
+	cacheObjLayer := newCacheObjectsFn()
+	if cacheObjLayer != nil {
+		cs := cacheObjLayer.StorageInfo(context.Background())
+		ch <- prometheus.MustNewConstMetric(
+			prometheus.NewDesc(
+				prometheus.BuildFQName("minio", "disk", "storage_bytes"),
+				"Total cached storage available for current Minio server instance",
+				nil, nil),
+			prometheus.GaugeValue,
+			float64(cs.Total),
+		)
+		ch <- prometheus.MustNewConstMetric(
+			prometheus.NewDesc(
+				prometheus.BuildFQName("minio", "disk", "storage_free_bytes"),
+				"Total free cached storage available for current Minio server instance",
```

Comment:
"Total free cached storage available for current Minio server instance" is confusing - "Total cache storage remaining for current Minio server instance"

Reply:
Perhaps i can change this to "Total cache consumed on current Minio server instance"

Reply:
in that case, you will be reporting Total - Free, and the user will have to do a mental calculation of available space..

Reply:
Not mental its available in UI @poornas its no different than currently sending total, free as separate values in CacheStorageInfo()

Reply:
Oh wait i might have said it wrong. it should  be "Total cache available on for current Minio server instance"

Reply:
Total cache available is much better.